src/pages/new/index.tsx

```diff
@@ -139,8 +164,15 @@ const CourseFeatureCard = ({resource, className}: any) => {
         <div className="flex flex-col items-center h-full">
           <div className="relative z-10 flex flex-col h-full justify-between  items-center sm:p-8 p-5">
             <div className="flex flex-col items-center">
-              <Image src={image} width={200} height={200} alt={title} />
-              <h2 className="text-xl font-bold min-w-full mt-4 sm:mt-14 mb-2 leading-tighter group-hover:underline">
+              {image && (
+                <Image
+                  src={get(image, 'src', image)}
```

Comment:
I'm not quite sure why I have to use `get` this way but it's what typescript wanted.. This is how we call get the image URL in the base card resource